src/index.js

```diff
@@ -173,36 +174,48 @@ function participantConnected(participant) {
 
   const videoElement = document.createElement('video');
   participantDiv.appendChild(videoElement);
-
   participants.appendChild(participantDiv);
 
-  participant.tracks.forEach(track => trackAdded(participant, track));
-  participant.on('trackAdded', track => trackAdded(participant, track));
-  participant.on('trackRemoved', track => trackRemoved(participant, track));
-  participant.once('disconnected', () => participantDisconnected(participant));
+  participant.tracks.forEach(publication => trackPublished(participant, publication));
+  participant.on('trackPublished', publication => trackPublished(participant, publication));
```

Comment:
This is instructional, although just setting listeners for "trackSubscribed" and "trackUnsubscribed" would be sufficient, too. I'm happy with this. Would be nice to handle the "trackUnpublished" event, too, I think.

Reply:
@markandrus Since we don't do anything with "trackUnpublished" here, do you want to just log the event?

Reply:
Yeah, just logging is fine 👍